frontend/src/_components/MultiSelectUser.jsx

```diff
@@ -38,21 +38,23 @@ function MultiSelectUser({
   );
 
   function renderCustom(props, option) {
+    const selectOption = () => {
+      onSelect([...selectedValues, option]);
+    };
+
     return (
       <div className={`item-renderer`}>
         <div>
           <input
             type="checkbox"
             // eslint-disable-next-line no-unused-vars
-            onClick={(e) => {
-              onSelect([...selectedValues, option]);
-            }}
+            onClick={selectOption}
```

Comment:
You should set event listener on a common parent of these elements. That's the simplest best solution.